src/ImageRunModal.jsx

```diff
@@ -908,6 +931,46 @@ export class ImageRunModal extends React.Component {
                             <Checkbox isChecked={this.state.runImage} id="run-image-dialog-start-after-creation"
                                       onChange={value => this.onValueChanged('runImage', value)} label={_("Start after creation")} />
                         </FormGroup>
+                        <Grid hasGutter md={6}>
+                            { this.props.userServiceAvailable && this.props.systemServiceAvailable &&
```

Comment:
Why is this guarded by both services being present?

Reply:
hmm actually it should most likely follow the cpuShares logic `this.state.owner === systemOwner`